src/memory/server/MemoryServer.ts

```diff
@@ -55,20 +55,20 @@ export class MemoryServer {
                  <head>
                      <meta charset="utf-8">
                      <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-                     <link href="${newPanel.webview.asWebviewUri(
-                         vscode.Uri.joinPath(
-                             context.extensionUri,
-                             'dist/MemoryBrowser.css'
-                         )
-                     )}" type="text/css" rel="stylesheet"></link>
+<link href="${newPanel.webview.asWebviewUri(
+            vscode.Uri.joinPath(
+                context.extensionUri,
+                'dist/MemoryBrowser-bundle.css'
```

Comment:
We need to avoid changing the name and the location of the css and js files as possible as we can, it is breaking down the components who are referencing cdt-gdb-vscode.

Reply:
I've introduced this change because there is an `MemoryBrowser.tsx` file as a sibling of `index.tsx` but not actually the entry point to the webview code. That means that a `tsc` compilation of the webview assets will produce a `MemoryBrowser.js` file itself, which I don't think the bundle should overwrite.

If you see an alternative that allows both `tsc` output and bundled output to coexist, I'd be happy to apply it.

Reply:
Never mind - they end up in different folders, so it's fine for them to have the same name.